action.yml

```
@@ -3,6 +3,11 @@ description: '🐶 Run rubocop with reviewdog on pull requests to improve code r
 author: 'mgrachev (reviewdog)'
 inputs:
   # Please maintain inputs in alphabetical order
+  fail_level:
+    description: |
+      Exit code 1 for reviewdog if it finds at least 1 issue with severity greater than or equal to the given level [none,any,info,warning,error].
```

```suggestion
      Exit code 1 for reviewdog if it finds at least 1 issue with severity greater than or equal to the given level [any,info,warning,error].
```
https://github.com/reviewdog/reviewdog/pull/1854/files